DOCS-888: prefix exclusion of versioning prevents replication (#893)
minio/docs: source/includes/common-replication.rst

Use :mc-cmd:`mc version info` to validate the versioning status of both the source and remote buckets. 
Use the :mc-cmd:`mc version enable` command to enable versioning as necessary.

> If you exclude a prefix or folder from versioning within the source bucket, MinIO cannot replicate objects within that folder or prefix.

Author: I'm waffling on including this at all, as it calls attention to versioning exclusion in the first place.

Reviewer: Unless we are likely to change "no versioning, no replication," my thought is better to be clear about it.

Author: Taking Daryl's suggestions with a tweak

```suggestion
MinIO cannot replicate objects in bucket prefixes that you excluded from versioning.
```
